packages/toolkit/query/package.json

```diff
@@ -4,16 +4,9 @@
   "description": "",
   "type": "module",
   "module": "../dist/query/rtk-query.legacy-esm.js",
-  "main": "../dist/query/cjs/index.js",
+  "main": "./../dist/query/rtk-query.modern.mjs",
   "types": "./../dist/query/index.d.ts",
-  "exports": {
```

Comment:
Dropping the `exports` field should be okay - a bundler that supports `exports` should look at the root-level `package.json`.